stories/setup/webpack.config.babel.js

```diff
@@ -1,58 +1,18 @@
-import 'core/polyfill';
-
-import path from 'path';
-
-import config from 'config';
-import autoprefixer from 'autoprefixer';
-import webpack from 'webpack';
-
-import { getClientConfig } from 'core/utils';
+import { getPlugins, getRules } from '../../webpack-common';
+import webpackConfig from '../../webpack.prod.config.babel';
 
 module.exports = {
+  ...webpackConfig,
   module: {
     rules: [
-      {
-        test: /\.svg$/,
-        use: [{ loader: 'svg-url-loader', options: { limit: 10000 } }],
-      },
-      {
-        test: /\.scss$/,
-        use: [
-          { loader: 'style-loader' },
-          { loader: 'css-loader', options: { importLoaders: 2 } },
-          {
-            loader: 'postcss-loader',
-            options: {
-              plugins: [
-                autoprefixer({
-                  browsers: ['last 2 versions'],
-                  grid: false,
-                }),
-              ],
-            },
-          },
-          { loader: 'sass-loader', options: { outputStyle: 'expanded' } },
-        ],
-      },
+      ...getRules({ bundleStylesWithJs: true, includeBabelLoader: false }),
```

Comment:
`bundleStylesWithJs` set to `true` is the default in the dev webpack config.

Reply:
yeh but I need to still call getRules to override babeloptions (unless I add the plugin I noted below to the root .babelrc file. Maybe that's an option too..)